modules/uid2IdSystem.js

```diff
@@ -6,13 +6,17 @@
  * @requires module:modules/userId
  */
 
-import { logInfo } from '../src/utils.js';
+import { logInfo, logWarn } from '../src/utils.js';
 import {submodule} from '../src/hook.js';
 import {getStorageManager} from '../src/storageManager.js';
 import {MODULE_TYPE_UID} from '../src/activities/modules.js';
 
+// RE below lint exception: UID2 and EUID are separate modules, but the protocol is the same and shared code makes sense here.
+// eslint-disable-next-line prebid/validate-imports
+import { Uid2GetId, Uid2CodeVersion } from './uid2IdSystem_shared.js';
+
 const MODULE_NAME = 'uid2';
-const MODULE_REVISION = `1.0`;
+const MODULE_REVISION = Uid2CodeVersion;
```

Comment:
If the uid2 needs to be replaced with the EUID in europe, why does the uid2 module continue to have a gvlid?

Reply:
and why is it diff in the two modules?

Reply:
I've removed the gvlid from the UID2 module and updated the EUID module to the gvlid for The Trade Desk (as operator of EUID).